hazelcast/src/main/java/com/hazelcast/internal/ascii/rest/HttpGetCommandProcessor.java

```diff
@@ -78,7 +79,12 @@ private void handleMap(HttpGetCommand command, String uri) {
         int indexEnd = uri.indexOf('/', URI_MAPS.length());
         String mapName = uri.substring(URI_MAPS.length(), indexEnd);
         String key = uri.substring(indexEnd + 1);
-        Object value = textCommandService.get(mapName, key);
+        Object value;
+        if(MAP_ENTRIES_COMMAND.equalsIgnoreCase(key)){
```

Comment:
If I read the surrounding code correctly you won't be able to fetch the value for the key `entries` of a map anymore. This maybe a corner case, but is at least very unexpected.